docker-compose/core-valid/config/core-valid-gridcapa-task-manager-application.yml

```diff
@@ -65,10 +65,11 @@ task-server:
       - REMEDIAL-ACTIONS-RESULT
       - REX-RESULT
     enable-export-logs: false
+    manual-upload-base-path: CORE/VALID
 
 minio-adapter:
   bucket: ${GRIDCAPA_MINIO_BUCKET:gridcapa}
-  base-path: ${GRIDCAPA_MINIO_BASE_PATH:/CORE/VALID}
+  base-path: ${GRIDCAPA_MINIO_BASE_PATH:CORE/VALID}
```

Comment:
Shouldn't the basepath be empty by default if no environment variable is found?

Reply:
in docker config the value isn't set ... but you are right could be removed